### DISCUSSION TITLE
What are possible ways to use tracked segments?

### DISCUSSION THREAD

**Discussion by ValentinaPanetta** [pimcore/pimcore#14888] [Q&A / Support] [answered]
I'm having a hard time with the following question from the Academy Senior Developer course "Targeting and Personalization".

**Question 7: What are possible ways to use tracked segments?**

- [ ] Tracked segments cannot be used > false
- [ ] Calculate target groups based on tracked segments. > true, it says so in the example usecases and in the videos
- [ ] Create full automatic personalization based on tracked segments. > true, there's a whole chapter about it
- [ ] Store tracked segments in customer objects > true, it's mentioned in the example usecases.

I tried various combinations and I still don't get it right. I'm very frustrated. Either the answer eludes me or there's a bug.

**Comment by noolldave** [comment]


**Reply by dvesh3** [reply]
@noolldave please write the question you are struggling with and your answers as the order of the questions are random.

**Reply by noolldave** [reply]
done

**Comment by twin-elements** [comment, marked as answer]
7. What are possible ways to use tracked segments?
a. Calculate target groups based on tracked segments.
b. Create full automatic personalization based on tracked segments.
c. Store tracked segments in customer objects.

**Reply by dvesh3** [reply]
Read all information about the answers:
https://pimcore.com/docs/platform/Customer_Management_Framework/Personalization/Additional_Targeting_Role_Components/
https://pimcore.com/docs/platform/Customer_Management_Framework/SegmentAssignment/
